### PR TITLE
Add Finnish translations

### PR DESCRIPTION
**Pull request by rizka10**
Add Finnish translations

### CODE REVIEW

File: config/locales/client.en.yml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-en:
```
Comment (ZogStriP) [inline config/locales/client.en.yml]: Why did you remove this file?

File: config/locales/client.fi.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+en:
```
Comment (ZogStriP) [inline config/locales/client.fi.yml]: This first line must match the locale (`fi` in your case)

Comment (rizka10) [inline config/locales/client.fi.yml]: I see. Well, the good news is that I obviously couldn't do persistent damage.

This pull request can be deleted (if it wasn't already). The whole Github feels like big maze to me, so I leave it for others until I gain some interest. I'll just keep translating what appears in Transifex

**Comment by ZogStriP** [comment]
Thanks @rizka10. I added the file manually ;)
